src/include/migraphx/op/multibroadcast.hpp

```diff
@@ -37,8 +37,13 @@ namespace op {
 /**
  * Broadcast multiple dimensions between two tensors.
  * Two versions of this operator: one input and two inputs.
- * One input version uses output_lens attribute and broadcasts to it.
- * Two inputs version broadcasts both inputs to the common shape at evaluation time.
+ * One input version uses output_lens attribute and broadcasts to it (does not support
+ * dynamic shape input)).
+ * Two inputs version broadcasts to the shape of the second input at evaluation time.
+ * Use the make_op() command with two shape arguments for dynamic input and the 1-argument version
+ * for static input, since later compiler passes will differentiate the two
+ * and optimize them differently; or use the add_common_op() method which wraps this
+ * decision logic.
```

Comment:
i don't think it is necessary to mention this here.

Reply:
I agree.

Reply:
How can we make it clear which wrapping function is to be used and when?